hadoop-hdds/common/src/main/java/org/apache/hadoop/utils/db/DBStoreBuilder.java

```diff
@@ -220,4 +227,15 @@ private File getDBFile() throws IOException {
     return Paths.get(dbPath.toString(), dbname).toFile();
   }
 
+  private File getWALFile() throws IOException {
+    if (walPath == null) {
+      LOG.error("Write-ahead-log path is " +
```

Comment:
Ok, in most cases this would not happen. No one would map a WAL to a different disk. If it is mapped, then we can probably log that we detected it is mapped. This is certainly not an error.